pom.xml

```
@@ -35,13 +35,15 @@ THE SOFTWARE.
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>postgresql-fingerprint-storage</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <packaging>pom</packaging>
+    <packaging>hpi</packaging>
 
     <properties>
         <jenkins.version>2.245</jenkins.version>
```

```suggestion
        <jenkins.version>2.248</jenkins.version>
```

You will need new APIs, right?

---

I am getting some troubles with 2.248. I am looking into resolving them.

---

Resolved. Maven had downloaded an empty 2.248 jar, which was causing this issue.